Base/usr/share/man/man1/clear.md

```diff
@@ -0,0 +1,13 @@
+## Name
+
+chres - Clear the terminal
```

Comment:
Is `chres` the correct name here? Should it be `clear` instead?

Reply:
Yes, fixed.